src/main/java/com/exadel/hotel/room/controller/RoomController.java

```diff
@@ -32,21 +33,23 @@ public ResponseEntity<List<RoomDto>> getRoomsByFloorId(@PathVariable("floorId")
 
     @PostMapping("/add")
     public ResponseEntity<RoomDto> add(@RequestBody RoomDto roomDto) {
-        return ResponseEntity.ok(roomService.add(roomDto));
+        return new ResponseEntity<>(roomService.add(roomDto), HttpStatus.CREATED);
     }
 
     @PutMapping("/update/{id}")
     public ResponseEntity<RoomDto> update(@PathVariable("id") Long id, @RequestBody RoomDto roomDto) {
-        return ResponseEntity.ok(roomService.update(id, roomDto));
+        return new ResponseEntity<>(roomService.update(id, roomDto), HttpStatus.ACCEPTED);
```

Comment:
I don't think ACCEPTED it's correct  you could check here: https://stackoverflow.com/questions/2342579/http-status-code-for-update-and-delete 
I think  here you can return 201